src/main/java/woowacourse/shoppingcart/dao/CustomerDao.java

```diff
@@ -32,10 +32,19 @@ public CustomerDao(final JdbcTemplate jdbcTemplate) {
         this.jdbcTemplate = jdbcTemplate;
     }
 
-    public Long findIdByUserName(final String userName) {
+    public Customer findById(final Long id) {
+        try {
+            final String query = "SELECT id, username, password, phone_number, address FROM customer WHERE id = ?";
+            return jdbcTemplate.queryForObject(query, customerRowMapper, id);
+        } catch (final EmptyResultDataAccessException e) {
+            throw new InvalidCustomerException();
+        }
+    }
+
+    public Long findIdByUserName(final String username) {
```

Comment:
```suggestion
    public Long findIdByUsername(final String username) {
```

그런데 테스트 코드에서만 사용되고 있군요 😢